spec/models/orchestration_stack/retirement_management_spec.rb

```diff
@@ -13,7 +13,7 @@
       @stack.retirement_check
       @stack.reload
       expect(@stack.retirement_last_warn).not_to be_nil
-      expect(Time.now.utc - @stack.retirement_last_warn).to be < 30
+      expect(Time.now.utc - @stack.retirement_last_warn).not_to be_nil
```

Comment:
@tinaafitz is this essentially the same check as [retirement_management_spec.rb +15](https://github.com/ManageIQ/manageiq/pull/17730/files#diff-11f68ca7512f1b64d41f0b295ea4ae96R15)